examples/user_guide/14-Large_Data.ipynb

```diff
@@ -544,11 +532,24 @@
   }
  ],
  "metadata": {
+  "kernelspec": {
+   "display_name": "Python 3",
+   "language": "python",
+   "name": "python3"
+  },
   "language_info": {
+   "codemirror_mode": {
+    "name": "ipython",
+    "version": 3
+   },
+   "file_extension": ".py",
+   "mimetype": "text/x-python",
    "name": "python",
-   "pygments_lexer": "ipython3"
+   "nbconvert_exporter": "python",
+   "pygments_lexer": "ipython3",
+   "version": "3.6.6"
   }
  },
  "nbformat": 4,
- "nbformat_minor": 1
+ "nbformat_minor": 2
```

Comment:
Just need to clear out this metadata.

Reply:
Why?

Reply:
This metadata is always there, your script doesn't strip this.

Reply:
Ah sorry, missed the bit above.

Reply:
Fixed.